src/lib/journey/journey.interfaces.ts

```diff
@@ -47,7 +50,7 @@ export interface JourneyStoreValue {
     callbacks: CallbackMetadata[];
     step: StepMetadata;
   } | null;
-  step: StepTypes;
+  step?: StepTypes;
```

Comment:
This was required because [here](https://github.com/ForgeRock/forgerock-web-login-framework/pull/259/files#diff-e01603033c52bf6a043d215ff082252220802c8f537131e59b6fc3eb8de26cd3R136) `step` can be null but the type is saying it must be there.

The alternative is to move it inside of the `if` statement above but I wasn't sure about that change completely.